Doc/library/datetime.rst

```diff
@@ -438,24 +501,27 @@ Other constructors, all class methods:
 
 .. classmethod:: date.fromordinal(ordinal)
 
-   Return the date corresponding to the proleptic Gregorian ordinal, where January
-   1 of year 1 has ordinal 1.  :exc:`ValueError` is raised unless ``1 <= ordinal <=
-   date.max.toordinal()``. For any date *d*, ``date.fromordinal(d.toordinal()) ==
-   d``.
+   Return the date corresponding to the proleptic Gregorian ordinal, where
+   January 1 of year 1 has ordinal 1.
+
+   :exc:`ValueError` is raised unless ``1 <= ordinal <=
+   date.max.toordinal()``. For any date *d*,
+   ``date.fromordinal(d.toordinal()) == d``.
 
 
 .. classmethod:: date.fromisoformat(date_string)
 
-  Return a :class:`date` corresponding to a *date_string* in the format emitted
-  by :meth:`date.isoformat`. Specifically, this function supports strings in
-  the format(s) ``YYYY-MM-DD``.
+   Return a :class:`date` corresponding to a *date_string* given in the format
+   ``YYYY-MM-DD``.
 
-  .. caution::
+      >>> from datetime import date
+      >>> date.fromisoformat('2019-12-04')
+      datetime.date(2019, 12, 4)
```

Comment:
Completely random remark, but I tend to try to never give dates examples with the day number below or equal to 12. That makes it unambiguous which one is which ... even if just above you have `YYYY-MM-DD`

Reply:
Good point.  I can do a sweep through to make sure that is enforced everywhere in this .rst.